sdcm/cluster.py

```diff
@@ -1966,16 +1966,21 @@ def install_manager_agent(self, package_path=None):
                 self.parent_cluster.params.get("scylla_mgmt_repo"))
             package_name = 'scylla-manager-agent'
         package_mgr = "yum" if self.distro.is_rhel_like else "apt-get"
+
+        @retrying(n=5, sleep_time=2, allowed_exceptions=UnexpectedExit)
+        def install_package():
+            self.remoter.run(f'sudo {package_mgr} install -y {package_name}')
```

Comment:
remoter have method sudo. it is same as run, but execute command with sudo

Reply:
you are right, changed that